tests/fixtures/webext-as-library/helpers.js

```diff
@@ -0,0 +1,15 @@
+const assert = require('assert');
+
+function testWebExtModuleImports(webExt) {
```

Comment:
Nit, personally I would rename the exported helper function to `testModuleExports` (mainly because we are testing what we do export when web-ext is loaded as a module).